org/docs/183.js

```diff
@@ -0,0 +1,2 @@
his}return l(t,e),t.prototype.provideCodeActions=function(e,t,n,r){return f(this,void 0,void 0,(function(){var r,i,o,s,a,u,c=this;return p(this,(function(l){switch(l.label){case 0:return r=e.uri,i=e.getOffsetAt({lineNumber:t.startLineNumber,column:t.startColumn}),o=e.getOffsetAt({lineNumber:t.endLineNumber,column:t.endColumn}),s=L._convertOptions(e.getOptions()),a=n.markers.filter((function(e){return e.code})).map((function(e){return e.code})).map(Number),[4,this._worker(r)];case 1:return[4,l.sent().getCodeFixesAtPosition(r.toString(),i,o,a,s)];case 2:return!(u=l.sent())||e.isDisposed()?[2,{actions:[],dispose:function(){}}]:[2,{actions:u.filter((function(e){return 0===e.changes.filter((function(e){return e.isNewFile})).length})).map((function(t){return c._tsCodeFixActionToMonacoCodeAction(e,n,t)})),dispose:function(){}}]}}))}))},t.prototype._tsCodeFixActionToMonacoCodeAction=function(e,t,n){for(var r=[],i=0,o=n.changes;i<o.length;i++)for(var s=0,a=o[i].textChanges;s<a.length;s++){var u=a[s];r.push({resource:e.uri,edit:{range:this._textSpanToRange(e,u.span),text:u.newText}})}return{title:n.description,edit:{edits:r},diagnostics:t.markers,kind:"quickfix"}},t}(L),T=function(e){function t(){return null!==e&&e.apply(this,arguments)||this}return l(t,e),t.prototype.provideRenameEdits=function(e,t,n,r){return f(this,void 0,void 0,(function(){var r,i,s,a,u,c,l,f,d,h;return p(this,(function(p){switch(p.label){case 0:return r=e.uri,i=r.toString(),s=e.getOffsetAt(t),[4,this._worker(r)];case 1:return[4,(a=p.sent()).getRenameInfo(i,s,{allowRenameOfImportPath:!1})];case 2:if(!1===(u=p.sent()).canRename)return[2,{edits:[],rejectReason:u.localizedErrorMessage}];if(void 0!==u.fileToRename)throw new Error("Renaming files is not supported.");return[4,a.findRenameLocations(i,s,!1,!1,!1)];case 3:if(!(c=p.sent())||e.isDisposed())return[2];for(l=[],f=0,d=c;f<d.length;f++)h=d[f],l.push({resource:o.Sf.parse(h.fileName),edit:{range:this._textSpanToRange(e,h.textSpan),text:n}});return[2,{edits:l}]}}))}))},t}(g);function N(e){P=W(e,"typescript")}function E(e){I=W(e,"javascript")}function K(){return new Promise((function(e,t){if(!I)return t("JavaScript not registered!");e(I)}))}function R(){return new Promise((function(e,t){if(!P)return t("TypeScript not registered!");e(P)}))}function W(e,t){var n=new u(t,e),r=function(){for(var e=[],t=0;t<arguments.length;t++)e[t]=arguments[t];return n.getLanguageServiceWorker.apply(n,e)},i=new b(r);return o.Mj.registerCompletionItemProvider(t,new y(r)),o.Mj.registerSignatureHelpProvider(t,new S(r)),o.Mj.registerHoverProvider(t,new w(r)),o.Mj.registerDocumentHighlightProvider(t,new k(r)),o.Mj.registerDefinitionProvider(t,new x(i,r)),o.Mj.registerReferenceProvider(t,new C(i,r)),o.Mj.registerDocumentSymbolProvider(t,new M(r)),o.Mj.registerDocumentRangeFormattingEditProvider(t,new D(r)),o.Mj.registerOnTypeFormattingEditProvider(t,new A(r)),o.Mj.registerCodeActionProvider(t,new O(r)),o.Mj.registerRenameProvider(t,new T(r)),new v(i,e,t,r),r}}}]);
```

Comment:
These are generated files, right? Why commit them? Maybe a git ignore update is required first.

Reply:
charts.org also includes files in docs.
I think this is used for deployment using github pages.